README.md

```diff
@@ -429,10 +432,47 @@ const { getByTestId } = render(
 expect(getByTestId('test')).not.toBeVisible();
 ```
 
+### `toHaveAccessibilityState`
+
+```ts
+toHaveAccessibilityState(state: {
+  disabled?: boolean;
+  selected?: boolean;
+  checked?: boolean | 'mixed';
+  busy?: boolean;
+  expanded?: boolean;
+});
+```
+
+Check that the element has given accessibility state entries. This matcher takes into account
+implied default accessibility state entries of `disabled: false`, `selected: false`, `busy: false`,
+matching the screen reader behaviour of React Native apps on both iOS and Android.
+
+This matcher is comptabile with `*ByRole` and `*ByA11State` queries from React Native Testing
+Library.
+
+#### Examples
+
+```js
+render(<View testID="view" />);
+expect(screen.getByTestId('view')).toHaveAccessibilityState({ disabled: false });
+expect(screen.getByTestId('view')).toHaveAccessibilityState({ selected: false });
+expect(screen.getByTestId('view')).toHaveAccessibilityState({ busy: false });
+expect(screen.getByTestId('view')).not.toHaveAccessibilityState({ checked: false });
+expect(screen.getByTestId('view')).not.toHaveAccessibilityState({ expanded: false });
```

Comment:
these 2 last examples confused me for a bit because we also have 
```js
expect(screen.getByTestId('view')).not.toHaveAccessibilityState({ checked: true });
expect(screen.getByTestId('view')).not.toHaveAccessibilityState({ expanded: true });
```
right?

Reply:
Thanks for pointing this out. I will add additional docs to explain it. Basically for `checked` & `expanded` having `false` value is considered something different than not having value. While for `disabled`, `selected` and `busy` not having value equal to having `false` value. 

Part of value proposition of this matcher is to encapsulate that behaviour.

Reply:
I've rewritten the docs, pls check if under current docs that makes more sense.

Reply:
Much clearer to me :)